libc/lib/CMakeLists.txt

```diff
@@ -51,7 +51,13 @@ foreach(archive IN ZIP_LISTS
       PROPERTIES
         OUTPUT_NAME ${archive_1}.bc
     )
-    list(APPEND added_gpu_bitcode_targets ${archive_1}bitcode)
+    list(APPEND added_bitcode_targets ${archive_1}bitcode)
+
+    # A dummy library required make the custom command that creates the bitcode
+    # actually build so it can be installed.
+    add_library(${archive_1}dummy STATIC stub.cpp)
```

Comment:
nit: can you rename this: `${archive_1}_stub` instead?

Reply:
Done